### PR TITLE
Add neurokit2

### PR DESCRIPTION
**Pull request by hoechenberger**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/neurokit2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{% set name = "NeuroKit2" %}
+{% set version = "0.0.42" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```
Comment (synapticarbors) [inline recipes/neurokit2/meta.yaml:9]: It looks like there is some capitalization issues. In the url, the package name should be `neurokit2`.

File: recipes/neurokit2/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
```
Comment (synapticarbors) [inline recipes/neurokit2/meta.yaml:20]: it looks like you'll need `pytest-runner` as well since it's listed in `setup_requires` in the `setup.py`.

Comment (hoechenberger) [reply recipes/neurokit2/meta.yaml:20]: Good catch, thanks! Let's see if it works now.

File: recipes/neurokit2/meta.yaml (hunk 1 above)
Comment (hoechenberger) [reply recipes/neurokit2/meta.yaml]: Fixed

**Comment by hoechenberger** [comment]
@synapticarbors It's building correctly now.

**Comment by hoechenberger** [comment]
Thanks!

**Pull request merged**